zellij-server/src/pty.rs

```diff
@@ -291,43 +293,43 @@ pub(crate) fn pty_thread_main(mut pty: Pty, layout: Box<Layout>) {
                 pty.bus
                     .senders
                     .send_to_screen(ScreenInstruction::GoToTab(tab_index, Some(client_id)))
-                    .unwrap();
+                    .with_context(err_context)?;
             },
             PtyInstruction::NewTab(terminal_action, tab_layout, tab_name, client_id) => {
                 pty.spawn_terminals_for_layout(
                     tab_layout.unwrap_or_else(|| layout.new_tab()),
                     terminal_action.clone(),
                     client_id,
-                );
+                )?;
```

Comment:
Is this an oversight?

Reply:
no, because I change `spawn_terminals_for_layout` to return `Result`

Reply:
Oh, I see, I'll fix that.